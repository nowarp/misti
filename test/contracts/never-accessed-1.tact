fun test(): Int {
    let a: Int = 20;
    if (true) { // error: write-only variable
        a = 42;
    }
    return a;
}
