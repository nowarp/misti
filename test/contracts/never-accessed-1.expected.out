test/contracts/never-accessed-1.tact:2:5:
  1 | fun test(): Int {
> 2 |     let a: Int = 20;
          ^
  3 |     if (true) { // error: write-only variable
Variable a is never accessed
